### PR TITLE
Added Order page Features

### PR DESCRIPTION
**Pull request by Anshita-Bhasin**
Hi Naveen,
Can you please check this PR?

- added order page 
- --added order test cases
Thanks
Anshita

### CODE REVIEW

File: src/Pages/OrderPage.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package Pages;
+
+public class OrderPage {
+
+
+
+    public static void doOrder()
+    {
+        System.out.println("Order Page");
```
Comment (naveenanimation20) [inline src/Pages/OrderPage.java:10]: where is order page actions/methods?
where is the logic?

File: src/tests/OrderPageTest.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package tests;
+
+public class OrderPageTest {
+
+
```
Comment (naveenanimation20) [inline src/tests/OrderPageTest.java:5]: where is @Test annotation?

File: src/tests/OrderPageTest.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    public void orderTest()
+    {
+        System.out.println(true);
```
Comment (naveenanimation20) [inline src/tests/OrderPageTest.java:9]: where is assertion logic?

**Review by naveenanimation20 — CHANGES_REQUESTED** [review]
Hi,
Please check my review comments and make the changes accordingly.

Thanks
Naveen

File: src/Pages/OrderPage.java (hunk 1 above)
Comment (Anshita-Bhasin) [reply src/Pages/OrderPage.java:10]: Done. Logic is added.

File: src/tests/OrderPageTest.java (hunk 2 above)
Comment (Anshita-Bhasin) [reply src/tests/OrderPageTest.java:5]: Added the logic.

File: src/tests/OrderPageTest.java (hunk 3 above)
Comment (Anshita-Bhasin) [reply src/tests/OrderPageTest.java:9]: Added.

**Review by naveenanimation20 — APPROVED** [review]
PR is approved.

**Comment by Anshita-Bhasin** [comment]
Order Page Changes Merging to the master branch.

**Pull request merged**